src/roadmap/common.py

```
@@ -165,6 +165,66 @@ async def get_allowed_host_groups(
+        return set()
+
+    identity = _decode_identity(x_rh_identity)
+    subject = kessel.subject_from_identity(identity, settings.kessel_principal_domain)
+    client = kessel.get_client(settings)
+
+    try:
+        workspace_ids = set(kessel.host_groups_for(client, subject))
+    except HTTPException:
+        raise
+    except Exception as err:
+        logger.error(f"Error querying Kessel for host groups: {err}", exc_info=True)
```

Ruff (0.16.1)

[warning] Logging `.exception(...)` should be used instead of `.error(..., exc_info=True)`

(G201)